openlibrary/templates/account/create.html

```diff
@@ -37,7 +37,7 @@ <h1>$_("Sign Up")</h1>
 $if ctx.user:
     $def user_link(): <a href="$ctx.user.key">$ctx.user.displayname</a>
     <p>$:_("You are already logged into Open Library as %(user)s.", user=str(user_link()))</p>
-    <p>$:_('If you\'d like to create a new, different Open Library account, you\'ll need to <a href="javascript:;" onclick="document.forms[\'logout\'].submit()">log out</a> and start the signup process afresh.')</p>
+    <p>$:_('If you\'d like to create a new, different Open Library account, you\'ll need to <a href="javascript:;" onclick="document.forms[1].submit()">log out</a> and start the signup process afresh.')</p>
```

Comment:
```suggestion
    <p>$:_('If you\'d like to create a new, different Open Library account, you\'ll need to <a href="javascript:;" onclick="document.forms[\'hamburger-logout\'].submit()">log out</a> and start the signup process afresh.')</p>
```
I think that referencing the logout form with an index could be problematic if we add more forms to the page (the wrong form could easily be submitted).  Instead, let's update the hamburger menu logout form such that it has the name `hamburger-logout` and access the form by name.

To do this, you'll also have to update two additional files:
1. https://github.com/internetarchive/openlibrary/blob/master/openlibrary/templates/lib/nav_head.html
  - In this file, we construct an object that contains all of the information needed to render our hamburger menu.  The logout link should be updated to include a `name` with `hamburger-logout` as its value.
  - This 'name' should be added here: https://github.com/internetarchive/openlibrary/blob/bc89c578396062216f85dd4d61e3f97e1a4120f6/openlibrary/templates/lib/nav_head.html#L12
3. https://github.com/internetarchive/openlibrary/blob/master/openlibrary/templates/lib/header_dropdown.html
  - This template renders the hamburger menu based on the `props` object that is passed in.  If the link includes the key `name`, then the name attribute should be added to the form.
  - The login link is rendered here: https://github.com/internetarchive/openlibrary/blob/master/openlibrary/templates/lib/header_dropdown.html#L60-L63

Reply:
Doing something like this will include the `name` in the form, if a name exists:
```
$ name = 'name=%s' % link['name'] if 'name' in link else ''
<form action="$(link['post'])" method="post" $name>
  <button $(tracker)>$(link["text"])</button>
</form>
```